release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -95,6 +95,18 @@ A new oc command line interface (CLI) flag, `--import-mode`, has been added to t
 == Ingress Node Firewall Operator is generally available
 Ingress Node Firewall Operator was made a technology preview in {product-title} 4.12. With this release, Ingress Node Firewall Operator is generally available. You can now configure firewall rules at the node level. For more information, see xref:../networking/networking-operators-overview.adoc#networking-operators-overview[Ingress Node Firewall Operator].
 
+[id="ocp-4-14-networking-kernal-network-pinning"]
+== Dynamic use of non-reserved CPUs for OVS
+
+With this release, the Open vSwitch (OVS) networking stack can dynamically use non-reserved CPUs by default in performance-tuned clusters with a CPU manager policy set to `static`. 
+This dynamic use of available, non-reserved CPUs maximizes compute resources for OVS and minimizes network latency for workloads during periods of high demand. 
+OVS remains unable to dynamically use isolated CPUs assigned to containers in `Guaranteed` QoS pods. This separation avoids disruption to critical application workloads. 
+
+[NOTE]
+====
+When the Node Tuning Operator (NTO) recognizes the performance conditions to activate the use of non-reserved CPUs, there is a few seconds delay while  `ovnkube-node` configures the CPU affinity alignment of OVS daemons running on the CPUs. During this window, if a `Guaranteed` QoS pod starts, it can experience a latency spike. 
```

Comment:
* Not critical, but I checked the Node Tuning Operator docs and it doesn't look like they really use the "NTO" abbreviation. And since you don't use it again in this section, I'd recommend removing it.
* "a few seconds delay" could probably be worded a bit better. Maybe "a several second delay"?
* Should it be clarified what `ovnkube-node` is?

```suggestion
When the Node Tuning Operator recognizes the performance conditions to activate the use of non-reserved CPUs, there is a several second delay while `ovnkube-node` configures the CPU affinity alignment of OVS daemons running on the CPUs. During this window, if a `Guaranteed` QoS pod starts, it can experience a latency spike. 
```